step-certificates/templates/generate-secrets.yaml

```diff
@@ -0,0 +1,74 @@
+{{- if and .Values.existingSecrets.enabled  .Values.registrationAuthority }}
+apiVersion: v1
+kind: Secret
+type: smallstep.com/private-keys
+metadata:
+  name: {{ include "step-certificates.fullname" . }}-secrets
+  namespace: {{ .Release.Namespace }}
+{{- end }}
+---
```

Comment:
This file is is totally new. 

In the secrets.yaml file we are managing this secret, so if we want something special it should be managed there.